lib/components/AlbumScreen/song_menu.dart

```diff
@@ -249,6 +317,11 @@ class _SongMenuState extends State<SongMenu> {
                                       ?.playbackOrderShuffledButtonLabel ??
                                   "Shuffling",
                         };
+                        final playbackSpeedTooltip =
+                            AppLocalizations.of(context)
+                                    ?.playbackSpeedButtonLabel(
```

Comment:
Just rely on `AppLocalizations` being initialized here, and use the `!` operator, no need to provide a fallback text. The existence of the actual token is checked during compile time.